package.json

```diff
@@ -28,7 +28,8 @@
     "build": "tsc",
     "test": "jest --config jestconfig.json -i",
     "prepare": "npm run build",
-    "lint": "tslint -p tsconfig.json"
+    "lint": "tslint -p tsconfig.json",
+    "generate-migration": "node dist/generate-migration.js"
```

Comment:
This is probably not going to work when this will be packed and used as npm package. 
See some tutorials on web for creating CLI and npm [documentation](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bin):